src/Microsoft.DotNet.Build.Tasks/PackageFiles/VersionTools.targets

```diff
@@ -6,6 +6,13 @@
     <BuildInfoCacheDir>$(ToolsDir)BuildInfoCache/</BuildInfoCacheDir>
   </PropertyGroup>
 
+  <!-- For backward compatibility, Include XmlUpdateSteps as Xml-type updaters. -->
```

Comment:
Can we file a tracking issue to do the work to move everyone onto the new plan and then remove this workaround?